src/components/spaces/show/fact_sidebar.js

```diff
@@ -0,0 +1,35 @@
+import React, {Component, PropTypes} from 'react'
+import {connect} from 'react-redux'
+
+import * as organizationActions from 'gModules/organizations/actions'
+import {findFacts} from 'gEngine/organization.js'
+
+import {FactList} from './list.js'
+
+function mapStateToProps(state) {
+  return {
+    organizations: state.organizations,
+    organizationFacts: state.facts.organizationFacts,
+  }
+}
+
+@connect(mapStateToProps)
+export class FactListContainer extends Component{
```

Comment:
Could this be a functional component instead of a class? Also, we'll want to be wary about re-renders of this component as it contains many histograms and is thus expensive to render.